.env.sample

```diff
@@ -4,4 +4,6 @@ FB_AUTHDOMAIN=<YOUR_FIREBASE_AUTHDOMAIN>
 FB_DATABASEURL=<YOUR_FIREBASE_DATABASEURL>
 FB_PROJECTID=<YOUR_FIREBASE_PROJECTID>
 FB_STORAGEBUCKET=<YOUR_FIREBASE_STORAGEBUCKET>
-FB_MESSAGINGSENDERID=<YOUR_FIREBASE_MESSAGINGSENDERID>
\ No newline at end of file
+FB_MESSAGINGSENDERID=<YOUR_FIREBASE_MESSAGINGSENDERID>
+LOGIN_API="http://localhost:8080/api/api/v1/auth/login"
```

Comment:
i think we should split the endpoint and the hostname ....
```js
AUTH_API_URL ="http://localhost:8080/api/api/v1/"
```
and then in your component that make the request  build the final endpoint 

```js
const furl = AUTH_API_URL + 'login';
```
Or use `axios` to buld the final URL.... check the [axios docs](https://github.com/axios/axios#request-config) (`baseURL` and `url`). 

On this way we can have a generic compornent to send http request ... i suppose there will be more URLs to hit ( yakchat-server) so we can reuse this generic component just creating a new instance with correct URL

Also... why the API endpoint has **/api/api/** ... ? just want to know if have to change something in the server's routers.... or it may be because your proxyserver and webpack?

Reply:
yes, that is a good idea i'm gonna change right now

Reply:
about the /api/api/ i leave a comment bellow